sources/packages/web/src/components/common/notes/Notes.vue

```diff
@@ -63,6 +63,7 @@ import CreateNoteModal from "@/components/common/notes/CreateNoteModal.vue";
 import { NoteBaseDTO, NoteDTO } from "@/types";
 import { ref } from "vue";
 import "@/assets/css/notes.scss";
+import "@/assets/css/common.scss";
```

Comment:
It is _common_ or related only to notes?

Reply:
common.scss can be imported into scss files that are used by our vue application like base.scss and from.io.scss so that it will come to notes implicitly.